Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "vss-client"
+name = "vss-client-ng"
```

Comment:
Might also want to make this change in the header of README.md, which should probably be named after the crate, not the repo.